config/generic-alligator-r2.cfg

```diff
@@ -0,0 +1,162 @@
+# This file contains common pin mappings for Alligator (Rev.2) board.
+# To use this config, the firmware should be compiled for the Arduino
+# Due.
+
+# Remember flash procedure:
+# sudo /etc/init.d/alligator-manager --erase
+# sudo bossac -e -w -v -b -R -i -p ttyAMA0 klipper.bin
+# See alligator github for alligator manager:
+# https://github.com/3Dartists/alligator-manager
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[static_digital_output DRV8825_microstepping]
+pins:PC10
+pins:PC29
+pins:PC19
+pins:PC18
+
+[dac084S085 my_digipot]
+enable_pin: PB14
+spi_bus : spi0
+motor1 : 100 # define motor current from 0 to 255 where 255 is 2.5A
```

Comment:
I think labelling these 'channel1' etc would be better (here and in Python module).
On the alligator they're motor1-4, but if used elsewhere the channel assignment is different (i.e. if channel1 is actually for motors1-4 [x,y,z1,z2] and channel2 for extruders [e1,e2]) then calling them motor1-4 would be very confusing.

Reply:
Thanks a lot for your suggestion! This is my first code on klipper and I really appreciate every comment. I hope the code is clearer that way

Reply:
It's more to maintain a separation between the device that your driver is referring to, and the one particular implementation of the device that is in your specific board.
So the DAC084S085 doesn't drive 'motors' right... it just drives a voltage output channel.  Your board just happens to use the first channel of voltage output for motor 1.. but another board might use the first channel for a laser power setpoint, or for a control valve % open command.
So it's important to remove the naming for the device away from the one implementation.

Good work, nicely done :)

Reply:
I understand what you mean, but the DAC on Alligator boards is only connected to the drivers. It cannot be used for anything else.

Thanks, Lorenzo

Reply:
But the DAC does exist in other places than the Alligator board ;)
Your driver is (supposed to be) for the DAC.. not **just** the DAC on the Alligator board.

Reply:
Of course, you are completely right, now using channel name seems more correct to me too. Thanks again for your feedback